src/Stratis.Features.FederatedPeg/Interfaces/IFederatedPegSettings.cs

```diff
@@ -17,7 +17,7 @@ public interface IFederatedPegSettings
         /// <summary>
         /// Ip Endpoints for the other nodes in the federation.
         /// </summary>
-        IEnumerable<IPEndPoint> FederationNodeIpEndPoints { get; }
+        SortedDictionary<IPEndPoint, IPEndPoint> FederationNodeIpEndPoints { get; }
```

Comment:
Why SortedDictionary? I assume the impact is trivial but do we ever need to access them in order?

Reply:
I agree, the federation itself should obviously be sorted, but I'm not sure I see the value in sorting their endpoints (the sorted key lookup degrades to O(logn) instead of O(1) for the normal dictionary).

Reply:
Done.